src/main/java/com/datalogics/pdf/security/LunaHsmManager.java

```diff
@@ -22,12 +22,17 @@
 
 /**
  * This class allows for connecting to a Luna SA Hsm Device.
+ *
+ * <p>
+ * Note: Once an instance of LunaHsmManager goes into a
+ * {@link com.datalogics.pdf.security.HsmManager.ConnectionState#DISCONNECTED} state during logout from the LunaSA you
+ * will need to create a new instance of this class if you need to login to the LunaSA device again.
```

Comment:
It's good to explain this, but I think this text should go on the Javadoc for the `HsmManager` interface: everything that implements the interface should work this same way.